index.js

```diff
@@ -103,4 +107,14 @@ spook.on('message', async (msg) => {
 	}
 });
 
+// This is a horrible way of sleeping, however I couldn't find a way to sleep with setInterval / setTimeout 
+function sleep(ms)
```

Comment:
I dont know if looping are bad for code performance, I might be wrong. But after searching through stackoverflow there is different method for sleep function by using promise.

```js
function sleep(ms) {
  return new Promise(resolve => setTimeout(resolve, ms));
}
```

But, I suggest you to using promise instead looping.

Reply:
Switched it to the resolving with a promise